sys/net/gnrc/netif/gnrc_netif.c

```diff
@@ -588,16 +588,16 @@ int gnrc_netif_ipv6_addr_add_internal(gnrc_netif_t *netif,
 
         gnrc_netif_ipv6_bus_post(netif, GNRC_IPV6_EVENT_ADDR_VALID, &netif->ipv6.addrs[idx]);
     }
-#if IS_ACTIVE(CONFIG_GNRC_IPV6_NIB_SLAAC)
-    else if (!gnrc_netif_is_6ln(netif)) {
+    else if (IS_USED(MODULE_GNRC_IPV6) &&
```

Comment:
Is checking for `MODULE_GNRC_IPV6` necessary? Wouldn't IPv6 support be always enabled when using `MODULE_GNRC_IPV6_NIB`?

Reply:
That would reintroduce a circular dependency again, as `gnrc_ipv6` requires the `gnrc_ipv6_nib` module.

Reply:
Ah yeah. Thanks for clarifying.